nucleus/security/core/src/main/java/com/sun/enterprise/security/auth/realm/certificate/CertificateRealm.java

```diff
@@ -101,7 +105,32 @@
 
     // Descriptive string of the authentication type of this realm.
     public static final String AUTH_TYPE = "certificate";
-    
+    public static final Map<String, String> oidMap;
+    static {
+        Map<String, String> oidMapInitialiser = new HashMap<>();
+        oidMapInitialiser.put(X500Name.commonName_oid.toString(), "CN");
```

Comment:
Tested using JDK11, and no complaints about CNFE owing from `sun.*` package, but might it be better to just directly map Strings rather than using these variables?

Reply:
Going to change it to directly using Strings - IBM JDK doesn't have the sun.* package space.